security/README.md

```diff
@@ -42,3 +42,47 @@ When requesting a security review for your application, please make sure you hav
 * [Unvalidated Redirects and Forwards](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.md)
 * [Web Service Security](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Web_Service_Security_Cheat_Sheet.md)
 * [XML Security](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/XML_Security_Cheat_Sheet.md)
+
+
+## Recommended Tools
```

Comment:
Please remove this section.

Reply:
Can you explain your reasoning here or maybe provide an alternative?

Reply:
I am curious too - looks good to me, but would like to understand the reasoning to understand what I a missing

Reply:
@jskulavik  I'm going to mark this as resolved, feel free to reopen